src/test/features/response/routes/reject-all-of-claim.ts

```diff
@@ -143,7 +143,7 @@ describe('Defendant response: full admission options', () => {
               .set('Cookie', `${cookieName}=ABC`)
               .send({ option: 'alreadyPaid' })
               .expect(res => expect(res).to.be.redirect
-                .toLocation(ResponsePaths.taskListPage
+                .toLocation(ResponsePaths.defendantHowMuchPaidClaimant
```

Comment:
Please add tests to cover redirect to `sendYourResponseByEmail` as well.

Reply:
Done